src/app/games/games.service.interface.ts

```diff
@@ -133,6 +163,29 @@ export interface ISubdivision {
   description: string;
 }
 
+/**
+ * Describes a game's shield output.
+ */
+export interface IShield {
+  /**
+   * Always 1.
+   */
+  schemaVersion: number;
```

Comment:
If this is always 1, and we aren't using it, I would be inclined to remove it.

Reply:
I mainly did that to have the struct be the same as the json.